README.md

```diff
@@ -73,6 +73,14 @@ versions of the compiler.
 [rustup.rs](https://www.rustup.rs/) may be used to manage
 installations of multiple versions of rust on single machine.
 
+The currently supported Rust version is nightly-2016-05-24.  To install
+it with rustup use the following:
```

Comment:
Ah indeed, I got the version from there as well if I remember correctly. I could submit a new PR with that if you want.